workspaces/src/worker/impls.rs

```diff
@@ -173,6 +189,77 @@ where
     }
 }
 
+#[cfg(feature = "experimental")]
+impl<T: ?Sized> Worker<T>
+where
+    T: NetworkClient,
+{
+    pub async fn changes_in_block(
```

Comment:
@ChaoticTempest Why are JSON RPC methods are exposed through "worker"? This looks like an abstraction leak to me.

Reply:
@ChaoticTempest kindly pinging you

Reply:
hmm, why would this be an abstraction leak? `Worker` here is just the interface where we want to surface RPC APIs we deem safe to do so, or APIs that are not directly tied to an account/contract that requires signing a transaction with. `Worker` here also protects against unwanted RPC methods by constraining certain RPC APIs to certain networks like sandbox's `fast_forward`. Where do you imagine it would live otherwise?